src/main/resources/db/migration/V5__give-select-grant-for-cloudsqliamuser.sql

```diff
@@ -0,0 +1,9 @@
+DO
+$$
+    BEGIN
+        IF (SELECT exists(SELECT rolname FROM pg_roles WHERE rolname = 'cloudsqliamuser'))
+        THEN
+            GRANT SELECT ON ALL TABLES IN SCHEMA veilarboppfolging TO cloudsqliamuser;
```

Comment:
Det er lurt å ha en `ALTER DEFAULT PRIVILEGUES` i tillegg, slik at man automatisk arver rettighetene for nye tabeller som blir opprettet.
Tror det blir noe a.la.
`ALTER DEFAULT PRIVILEGUES FOR ROLE cloudsqliamuser IN SCHEMA veilarboppgfolging GRANT SELECT ON ALL TABLES ;`